iocore/net/ActionProcessor.cc

```diff
@@ -0,0 +1,53 @@
+/** @file
```

Comment:
Hmmm, why is this a Processor? Processors have pretty significant semantics in our code, such that a Processor is heavily involved with the event processing. This is not that, is it ?